common/primitives/src/utils.rs

```diff
@@ -38,6 +40,23 @@ pub mod as_string {
 	}
 }
 
+/// Handle serializing and deserializing from `Vec<u8>` to a UTF-8 string
+#[cfg(feature = "std")]
+pub mod as_model {
+	use super::*;
+	use serde::{ser::Error, Deserialize, Deserializer, Serialize, Serializer};
+
+	/// Serializes a `Parquet Model` into a UTF-8 string
```

Comment:
This will likely just complicate things, but is it worthwhile to serialize into SCALE for storage?

I would guess that would just be more work on chain as the upload is in JSON, but I could imagine that we deserialize the JSON and serialize into SCALE for storage. Then RPC deserializes out of SCALE.

Call -> Storage -> RPC
JSON -> SCALE -> Object in JSON response

Reply:
hmm, i see the upside, but im not immediately sure what this would entail. i can investigate this.

Reply:
This would be an even larger switch up, but just saw this today: https://github.com/paritytech/scale-value

Reply:
As discussed, we'll table this for now.